### DISCUSSION TITLE
MRT 2 SAT

### DISCUSSION THREAD

**Discussion by pukozs75** [iptv-org/community#792] [Q&A]
### Channel Name

MRT 2 SAT

### Channel ID

MRT2Sat.mk

### Website

https://play.mrt.com.mk/live/46

### Notes

TV Channel from North Macedonia.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by Sphinxroot** [comment]
seem vod playlist token  ip enforcement  30 minute access key
https://vod-c57.interspace.com/channel_abr/46/playlist.m3u8?wmsAuthSign= +base64 token = (server_time=5/30/2022 2:33:01 PM&hash_value=xxxxxxxxx &validminutes=30&id=xxxxxxx)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19791
